main-app/app/scripts/gameboard.js

```diff
@@ -0,0 +1,31 @@
+myApp = angular.module('Tombola.MyModule', []);
+
+myApp.controller('MyController', function ($scope) {
+    var currentPlayer;
+    currentPlayer = '1';
+    $scope.gameBoard = '000000000';
+    $scope.Player1 = 'player1';
+    $scope.Player2 = 'player2';
+
+    $scope.chooseBlock = function (index){
+        if ($scope.gameBoard.charAt(index)!=="0"){
+            return;
+        }
+        if (currentPlayer === '1'){
+            $scope.gameBoard= setCharAt($scope.gameBoard ,index,'1');
+            document.getElementsByTagName('img')[index].setAttribute("src", "images/Crosses.gif");
```

Comment:
nope - do not manipulate the DOM directly like this. Angular will not track the changes. If this is stub code, say so, but binding is easier.